qa/integration-tests/src/test/java/io/camunda/it/client/UserTaskQueryTest.java

```diff
@@ -141,26 +146,31 @@ public void shouldRetrieveTaskByMapProcessInstanceVariable() {
 
   @Test
   public void shouldRetrieveTaskByProcessInstanceAndLocalVariable() {
-    final UserTaskVariableFilterRequest variableValueFilter =
-        new UserTaskVariableFilterRequest().name("task02");
+    final UserTaskVariableFilterRequest variableValueFilter1 =
+        new UserTaskVariableFilterRequest()
+            .name("task02")
+            .value(new StringPropertyImpl().eq("1").build());
+    final UserTaskVariableFilterRequest variableValueFilter2 =
+        new UserTaskVariableFilterRequest()
+            .name("task02")
+            .value(new StringPropertyImpl().eq("1").build());
```

Comment:
🔧 You could change this to `exists` instead of using the same `eq` query twice, I believe. That should return the initial behavior intended here in this use case.